bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/firmware/ProgressCallback.java

```diff
@@ -74,4 +75,14 @@
      * Callback operation to indicate that the firmware update was canceled.
      */
     void canceled();
+    
+    /**
+     * Callback operation to indicate that the percentage progress of the firmware update has changed. 
```

Comment:
As a binding developer it might be not clear enough how to use this interface after this operation has been introduced... So there are several mechanism now to inform about an update progress.... either using next operation or setPercentageProgress... furthermore it is not clear how this operation corresponds to the defineSequence operation